src/types/PickupItem.ts

```diff
@@ -0,0 +1,8 @@
+import { FoodType } from "./FoodType";
+import { QuantityUnity } from "./QuantityUnity";
+
+export interface PickupItem {
+    food: FoodType;
+    unity: QuantityUnity;
```

Comment:
`unity` -> `unit`

Reply:
Done (all the other places where this happens too)